miqsel/server.py

```diff
@@ -64,18 +64,23 @@ def vnc(self):
             else None
         )
 
-    def start(self):
+    def start(self, **kwargs):
         """Start selenium container"""
 
         img = self.cfg["container"]["image"]
         name = self.cfg["container"]["name"]
+        mount_dir = self.cfg["container"]["data_dir"]
+
+        if mount_dir != "None":
+            kwargs.update({"volumes": {mount_dir: {"bind": "/usr/tmp", "mode": "ro"}}})
```

Comment:
Strongly Suggest use either `/tmp` or `/var/tmp`

Reply:
`/tmp` in heavy container used for creating temp files and .x11 stuff. so its not valid.
about `/var/tmp` we can use but I think its sharing same like `/urs/tmp` right?

Reply:
Yes. @digitronik you are right, so instead of `/usr/tmp` use directly `/var/tmp`.